x-pack/test_serverless/api_integration/test_suites/observability/infra/infra.ts

```diff
@@ -114,30 +92,31 @@ export default function ({ getService }: FtrProviderContext) {
               ],
               metrics: [
                 {
-                  name: 'rx',
-                  value: 133425.6,
+                  name: 'rxV2',
+                  value: 17886.18845261874,
                 },
                 {
-                  name: 'tx',
-                  value: 135892.3,
+                  name: 'txV2',
+                  value: 18216.85858680644,
                 },
                 {
                   name: 'memory',
                   value: 0.9490000000000001,
                 },
                 {
-                  name: 'cpu',
-                  value: 1.021,
+                  name: 'cpuV2',
+                  value: 0.124,
                 },
                 {
                   name: 'diskSpaceUsage',
-                  value: 0,
+                  value: null,
                 },
                 {
                   name: 'memoryFree',
                   value: 1753829376,
                 },
               ],
+              monitored: true,
```

Comment:
Can we add the `monitored` flag also to the readme? 

A bit of a nit but `monitored` can be a bit misleading as all the hosts are "monitored" just not all use system integration. Maybe we can use `systemMetricsEnabled` or `hasSystemMetrics` or something around that.

Reply:
Makes sense. I'll change it to a more meaningful name.